crates/biome_css_formatter/tests/specs/scss/expression/parenthesized-binary.scss

```
@@ -1 +1,17 @@
-$value:(1 > $very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-long-var);
+$parenthesized:(1 > $very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-long-var);
+
+$parenthesized-tight-after:(1 >$very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-very-long-var);
```

Stylelint (17.11.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)